rfc047-release-process.md

```diff
@@ -44,40 +44,51 @@ Additional labels for pre-release and build metadata are available as extensions
 Chef Client specific clarifications:
 
 ```
-Given a version number MAJOR.MINOR.PATCH, increment the:
+Given a version number MAJOR.MINOR.BUILD:
 
-MAJOR version when the release will break existing functionality
-MINOR version when you add any new features
-PATCH version when you are only fixing bugs, typically regressions
+Increment the MAJOR version when the release will break existing functionality
+Increment the MINOR version when you add any new features
+The BUILD version is increased automatically by the build system
 ```
 
-##### Prerelease versions
+#### Continuous Integration / Continuous Delivery (build system)
+
+The BUILD version is numeric and must be increased for each build.
+
+The build system may consume unreleased BUILD versions. For example, a build which is successful but does not pass integration testing may not have its version reused.
```

Comment:
Yeah, there's maybe a line in there where you could reuse a build number for failed builds. But I guess we should just say they must all only be used one by the system that generates them, since it's going to generate them pretty close to when it's going to use them for something.